spec/hardware/cpu.fmf

```diff
@@ -97,6 +100,11 @@ example:
     cpu:
         cores: ">= 4"
 
+  - |
+    # Request a CPU supports hyper-threading.
```

Comment:
```suggestion
    # Request a CPU with hyper-threading enabled.
```

Small nuance shouldn't it be with hyper-threading `enabled` rather than just supported?